tests/installation/ipxe_install.pm

```diff
@@ -239,7 +239,7 @@ sub run {
             if (check_var('VIDEOMODE', 'text') and is_sle('<=15-SP5')) {
                 enter_cmd_slow('DISPLAY= yast.ssh');
             }
-            else {
+            elsif (check_var('VIDEOMODE', 'text')) {
```

Comment:
or `VIDEOMODE='ssh-x'`, it require typing "yast.ssh" too.

Reply:
Updated, VR is running.

Reply:
LGTM